client/analytics/report/revenue.js

```diff
@@ -78,34 +95,81 @@ class RevenueReport extends Component {
 				</a>
 			);
 			return [
-				row.start_date,
-				orderLink,
-				formatCurrency( gross_revenue ),
-				formatCurrency( refunds ),
-				formatCurrency( coupons ),
-				formatCurrency( taxes ),
-				formatCurrency( shipping ),
-				formatCurrency( net_revenue ),
+				{
+					display: <input type="checkbox" />,
+					value: false,
+				},
+				{
+					display: formatDate( 'm/d/Y', row.date_start ),
```

Comment:
datepicker uses for i18n 
```js
moment( row.date_start ).format( __( 'MM/DD/YYYY', 'wc-admin' ) );
```

Reply:
I like the idea of translating the date format, but I'm using the `@wordpress/date` `format`, which uses PHP-style date formatting (I tried this, and got `AprApr/SunSun/2018201820182018`). Are you able to use `@wordpress/date` in the datepicker, so that we can use the same string for translation?

Reply:
>Are you able to use @wordpress/date in the datepicker

I tried this route and ran into problems with i18n, specifically localizing `m/d/Y`. One option is to grab the PHP-style date format from `wp.date.getSettings().formats.date`. This is the way the site owner designates how they like dates displayed on the public facing side. 

In this case, I think this approach will work because it makes sense in the context of dates displayed in the table. For the datepicker it doesn't work, specifically date formats for strings entered by the user via keyboard. Anyways, feel free to disregard my comment.


>I tried this, and got `AprApr/SunSun/2018201820182018`

hmmm, weird. Moment's format is giving me something more expect

Reply:
Oh, getting the site's date format sounds like a good idea 👍 

> Moment's format is giving me something more expected

Yeah, you're using moment.format – `format` here is from wp.date, which is a different function that [uses the PHP-style date format-strings](https://github.com/WordPress/gutenberg/blob/master/packages/date/src/index.js#L277).